Formula/opencv.rb

```diff
@@ -24,8 +25,13 @@ class Opencv < Formula
   depends_on "tbb"
 
   resource "contrib" do
-    url "https://github.com/opencv/opencv_contrib/archive/4.0.1.tar.gz"
-    sha256 "0d8acbad4b7074cfaafd906a7419c23629179d5e98894714402090b192ef8237"
+    url "https://github.com/opencv/opencv_contrib/archive/4.1.0.tar.gz"
+    sha256 "e7d775cc0b87b04308823ca518b11b34cc12907a59af4ccdaf64419c1ba5e682"
+  end
+
+  patch :p1 do
```

Comment:
`:p1` is the default, so it can be removed.

```
==> Downloading https://github.com/opencv/opencv/pull/14308.patch?full_index=1
==> Downloading from https://patch-diff.githubusercontent.com/raw/opencv/opencv/pull/14308.patch?full_index=1
Warning: Patch reports different SHA256: 4d2ccaa710804eee5c2f8b729215aaeb995ae324487c1761df9a07812f02fbc5
Downloaded to: /Users/brew/Library/Caches/Homebrew/downloads/d147bd546d02183983a13d3ec79f058102fe6d68dd2220f090de8a2b9df99d02--14308.patch
SHA256: 9de405097cbafeadf0b980083edf1ff6b1a18e452bffea2b88a235c36677a1f8
```